### PR TITLE
[ci][jenkins]Uncomment integration traffic tests and fix routing issue for traffic…

### PR DESCRIPTION
**Pull request by 119Vik**
Signed-off-by: Vitalii <vitalij.kostenko@gmail.com>

## Summary
[skipJenkins] 
This PR removes "comment integration tests" instruction and fix routing for traffic tests

## Test Plan

Run Jenkins tests for the PRs

## Additional Information

added according to the @ssanadhya request

### CODE REVIEW

**Comment by 119Vik** [comment]
ReRunOAITests

**Comment by ssanadhya** [comment]
Please increase the timeout on `make integ_test` step to 45 minutes, as the timer expiry test cases add a wait time of 18-20 minutes.

File: ci-scripts/JenkinsFile-GitLab [hunk 1]
```diff
@@ -310,32 +311,32 @@ pipeline {
 
                 echo "Starting the integration Tests - S1AP Tester"
                 // We have removed the traffic testcases from mandatory suite.
-                timeout (time: 20, unit: 'MINUTES') {
+                timeout (time: 45, unit: 'MINUTES') {
                   myShCmdWithLog('cd lte/gateway && vagrant ssh magma_test -c "cd magma/lte/gateway/python/integ_tests/ && source ~/build/python/bin/activate && make integ_test"', 'archives/magma_run_s1ap_tester.log', new_host_flag, new_host_user, new_host)
                 }
                 // Adding a few tests not in the mandatory suite and the UL traffic scenarios (launched manually)
-                timeout (time: 45, unit: 'SECONDS') {
+                timeout (time: 45, unit: 'MINUTES') {
```
Comment (ssanadhya) [inline ci-scripts/JenkinsFile-GitLab]: We can remove lines 317-331 as these two tests (s1aptests/test_attach_ul_udp_data.py and s1aptests/test_attach_ul_tcp_data.py ) are included in mandatory tests suite.

File: ci-scripts/JenkinsFile-GitLab [hunk 2]
```diff
@@ -310,32 +311,32 @@ pipeline {
...
                   try {
                     myShCmdWithLogAppend('cd lte/gateway && vagrant ssh magma_test -c "cd magma/lte/gateway/python/integ_tests/ && source ~/build/python/bin/activate && make integ_test TESTS=s1aptests/test_attach_ul_tcp_data.py"', 'archives/magma_run_s1ap_tester.log', new_host_flag, new_host_user, new_host)
                   } catch (Exception e) {
                     echo "s1aptests/test_attach_ul_tcp_data testcase may fail"
                   }
                 }
-                timeout (time: 45, unit: 'SECONDS') {
+                timeout (time: 45, unit: 'MINUTES') {
```
Comment (ssanadhya) [inline ci-scripts/JenkinsFile-GitLab]: This is a single test run, so timing out in 45 seconds is the correct way to go.

File: ci-scripts/JenkinsFile-GitLab [hunk 3]
```diff
@@ -310,32 +311,32 @@ pipeline {
...
                   try {
                     myShCmdWithLogAppend('cd lte/gateway && vagrant ssh magma_test -c "cd magma/lte/gateway/python/integ_tests/ && source ~/build/python/bin/activate && make integ_test TESTS=s1aptests/test_attach_dl_udp_data.py"', 'archives/magma_run_s1ap_tester.log', new_host_flag, new_host_user, new_host)
                   } catch (Exception e) {
                     echo "s1aptests/test_attach_dl_udp_data testcase may fail"
                   }
                 }
-                timeout (time: 45, unit: 'SECONDS') {
+                timeout (time: 45, unit: 'MINUTES') {
```
Comment (ssanadhya) [inline ci-scripts/JenkinsFile-GitLab]: Same comment as above.

File: ci-scripts/JenkinsFile-GitLab [hunk 4]
```diff
@@ -260,7 +260,7 @@ pipeline {
             script {
                 myShCmdWithLog('cd lte/gateway && vagrant destroy --force magma_test && vagrant up magma_test', 'archives/magma_vagrant_test_up.log', new_host_flag, new_host_user, new_host)
                 // The Traffic Testcases are disabled from the Mandatory suite --> will be run manually
-                myShCmd('sed -i -f ci-scripts/removeLastUlTestCases.sed lte/gateway/python/integ_tests/defs.mk', new_host_flag, new_host_user, new_host)
+                // myShCmd('sed -i -f ci-scripts/removeLastUlTestCases.sed lte/gateway/python/integ_tests/defs.mk', new_host_flag, new_host_user, new_host)
```
Comment (ssanadhya) [inline ci-scripts/JenkinsFile-GitLab]: Remove commented code.

**Review by ssanadhya — CHANGES_REQUESTED** [review]

**Comment by ssanadhya** [comment]
@119Vik Looking at the failure here http://54.184.131.175:8080/blue/organizations/jenkins/Converged_MME_Pipeline/detail/Converged_MME_Pipeline/1088/pipeline/717, you need to remove the route added for noS11 case before adding the one for S11 setup. The tests are still green as route only impacts the traffic tests, which are not running for S11 setup at the moment.

**Comment by 119Vik** [comment]
RestartIntegrationTests

**Comment by 119Vik** [comment]
RestartIntegrationTests

**Comment by 119Vik** [comment]
RestartIntegrationTests

**Comment by 119Vik** [comment]
RestartIntegrationTests

File: ci-scripts/JenkinsFile-GitLab [hunk 5]
```diff
@@ -260,7 +260,6 @@ pipeline {
             script {
                 myShCmdWithLog('cd lte/gateway && vagrant destroy --force magma_test && vagrant up magma_test', 'archives/magma_vagrant_test_up.log', new_host_flag, new_host_user, new_host)
                 // The Traffic Testcases are disabled from the Mandatory suite --> will be run manually
```
Comment (rdefosse) [inline ci-scripts/JenkinsFile-GitLab]: Remove all that comment

File: ci-scripts/JenkinsFile-GitLab [hunk 6]
```diff
@@ -310,7 +310,7 @@ pipeline {
 
                 echo "Starting the integration Tests - S1AP Tester"
                 // We have removed the traffic testcases from mandatory suite.
-                timeout (time: 20, unit: 'MINUTES') {
+                timeout (time: 30, unit: 'MINUTES') {
                   myShCmdWithLog('cd lte/gateway && vagrant ssh magma_test -c "cd magma/lte/gateway/python/integ_tests/ && source ~/build/python/bin/activate && make integ_test"', 'archives/magma_run_s1ap_tester.log', new_host_flag, new_host_user, new_host)
                 }
                 // Adding a few tests not in the mandatory suite and the UL traffic scenarios (launched manually)
```
Comment (rdefosse) [inline ci-scripts/JenkinsFile-GitLab]: All these manual test launches SHALL be removed if they are now in the mandatory list

**Review by rdefosse — CHANGES_REQUESTED** [review]

File: ci-scripts/JenkinsFile-GitLab (hunk 5 above)
Comment (119Vik) [reply ci-scripts/JenkinsFile-GitLab]: Fixed

File: ci-scripts/JenkinsFile-GitLab (hunk 6 above)
Comment (119Vik) [reply ci-scripts/JenkinsFile-GitLab]: fixed

**Review by ssanadhya — APPROVED** [review]

**Review by rdefosse — APPROVED** [review]

**Pull request merged**